src/analytics/lambdas/clear-expired-events-workflow/check-clear-status.ts

```diff
@@ -53,13 +53,12 @@ export const handler = async (event: ClearExpiredEventsEvent) => {
     };
   } else if (response.Status == StatusString.FAILED || response.Status == StatusString.ABORTED) {
     logger.info(`Executing ${queryId} status of statement is ${response.Status}`);
-    const queryResult = await queryClearLog(appId);
```

Comment:
Is the sp log unavailable?

Reply:
want to keep log when Status == FAILED/ABORTED, queryClearLog will clean the error log